schemas/context/webpage.schema.json

```diff
@@ -0,0 +1,103 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/webpage",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Web Page",
+  "type": "object",
+  "description":
+    "Details about the web page where a web interaction occurred, as recorded by an `ExperienceEvent`.",
+  "definitions": {
+    "webpage": {
+      "properties": {
+        "xdm:siteSection": {
+          "title": "Site Section",
+          "type": "string",
+          "description":
+            "The normative name of the site section where this web page resides, which may be used to classify or categorize the interaction"
+        },
+        "xdm:name": {
+          "title": "Name",
+          "type": "string",
+          "description":
+            "The normative name of the web page. This name is not necessarily the page title or directy associate with page content, but is used to organize a site's pages for classification purposes"
+        },
+        "xdm:URL": {
+          "title": "URL",
+          "type": "string",
+          "format": "uri",
+          "description":
+            "The normative or usual URL of the web page.  This may or may not be the actual URL used to reach the page, which would be recorded using `WebLink`."
+        },
+        "xdm:server": {
+          "title": "Server",
+          "type": "string",
+          "format": "hostname",
+          "description":
+            "The normative or usual server that hosts the web page.  This may or may not be the host or server that actually served the page interaction, but is used for classification purposes."
+        },
+        "xdm:isErrorPage": {
+          "title": "Is Error Page",
+          "type": "boolean",
+          "description":
+            "Flag that indicate if the page is error page or not.  Error here is defined by the application, and may nor may not correspond to a page served with an HTTP error code.  This flag is used to broadly categorize web interactions."
+        },
+        "xdm:isHomePage": {
+          "title": "Is Home Page",
+          "type": "boolean",
+          "description":
+            "Flag that indicate if the page is the site home page or not.  The definition of home page is determined by the application, but is commonly used to designate a top level landing page or common site entry point.  This flag is used to broadly categorize web interactions."
+        },
+        "xdm:referrer": {
```

Comment:
Making `xdm:referrer` part of `webpage` is very counterintuitive. I think `webpage` should model the page where something is happening. How someone got there is a different container (and `weblink` looks very much like the correct container)

Reply:
I agree with your point about the incongruence of referrer in webpage. Lets put it in either the webLink or maybe more preferably in a peer to webpage and weblink.